### PR TITLE
Integrated smart contracts into sim repo with jupyter notebook & ape.

### PR DESCRIPTION
**Pull request by villanuevawill**
Done. Readme attached.

### CODE REVIEW

**Comment by villanuevawill** [comment]
image

File: notebook/apeworx.ipynb [hunk 1]
```diff
@@ -0,0 +1,455 @@
+{
```
Comment (dpaiton) [inline notebook/apeworx.ipynb:1]: You'll need to "Clear all outputs", save, then push up again to pass CI.

File: notebook/contracts/IERC20.sol [hunk 2]
```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
```
Comment (dpaiton) [inline notebook/contracts/IERC20.sol:1]: Can you move your `notebook` folder into `examples/notebooks` pls?

Comment (villanuevawill) [reply notebook/contracts/IERC20.sol:1]: Unfortunately it won't work this way.

Comment (villanuevawill) [reply notebook/contracts/IERC20.sol:1]: Current tooling needs everything in a notebook folder. We can update their core repo, but I think it makes more sense to have our notebooks in a notebook folder anyways.

**Review by sentilesdal — APPROVED** [review]
lgtm

**Pull request merged**